### PR TITLE
Feature/docker cli plugin

### PR DESCRIPTION
**Pull request by houdini91**

### CODE REVIEW

File: docker-cli-plugin/README.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
+# Scribe Docker CLI plugins 
+Scribe offers Docker CLI plugins for embedding evidence collecting and integrity verification to your workflows. \
+Actions are are wrappers to provided CLI tools.
+Plugins allow you to generate SBOMS as w
+
+## Install
+Scribe install script will install scribe docker CLI scripts 
+```
+curl -sSfL https://raw.githubusercontent.com/scribe-security/misc/master/docker-cli-plugin/install.sh | sh
```
Comment (ScribeEitan) [inline docker-cli-plugin/README.md:9]: I don't know if it's because it's not merged yet, but when I ran this curl I got 404

**Review by ScribeEitan — CHANGES_REQUESTED** [review]
left you an inline comment about the curl - please address it before I approve.

File: docker-cli-plugin/README.md (hunk 1 above)
Comment (houdini91) [reply docker-cli-plugin/README.md:9]: Replace the master with the name of the branch

Comment (ScribeEitan) [reply docker-cli-plugin/README.md:9]:
```
scribe info Installer - Scribe docker cli plugins
scribe info Selected, tool=gensbom, plugin=docker-bom
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
Warning: Failed to create the file /home/eitan/.docker/cli-plugins/docker-bom: 
Warning: No such file or directory
100   647  100   647    0     0   1350      0 --:--:-- --:--:-- --:--:--  1347
curl: (23) Failure writing output to destination
chmod: cannot access '/home/eitan/.docker/cli-plugins/docker-bom': No such file or directory
scribe info Installed /home/eitan/.docker/cli-plugins/docker-bom
scribe info Selected, tool=gensbom, plugin=docker-verify
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
Warning: Failed to create the file 
Warning: /home/eitan/.docker/cli-plugins/docker-verify: No such file or 
Warning: directory
100   650  100   650    0     0   1323      0 --:--:-- --:--:-- --:--:--  1323
curl: (23) Failure writing output to destination
chmod: cannot access '/home/eitan/.docker/cli-plugins/docker-verify': No such file or directory
scribe info Installed /home/eitan/.docker/cli-plugins/docker-verify
```

**Comment by houdini91** [comment]
Not sure what comment you are suggesting. 
Curl in readme always points to master obviously you can use other branches ..

**Comment by ScribeEitan** [comment]
it's ok, I didn't know it would add a different comment to the PR...

**Pull request merged**